### PR TITLE
Update dependencies and run tests for SF4

### PR DESCRIPTION
**Pull request by kosdfol**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ matrix:
           env: SYMFONY_VERSION=3.0.x
         - php: 7.0
           env: SYMFONY_VERSION=3.1.x
+        - php: 7.1
+          env: SYMFONY_VERSION=4.0.x
```
Comment (xabbuh) [inline .travis.yml]: For this version, the minimum stability needs to be enforced to be beta as there is no stable release yet. You can best do that probably by also defining a stability env var here like this:

```yaml
- php: 7.1
  env: SYMFONY_VERSION=4.0.x stability=beta
```

And then enforce this before the installation using the `composer config` command like this:

```yaml
install:
  - if [ "$stability" != "" ]; then composer config minimum-stability $stability; endif
  - if [ "$dependencies" = "lowest" ]; then composer update --prefer-lowest --prefer-stable --prefer-dist --no-interaction; else composer update --prefer-dist --no-interaction; fi;
```

File: .travis.yml [hunk 2]
```diff
@@ -32,12 +32,15 @@ matrix:
...
           env: SYMFONY_VERSION=3.1.x
+        - php: 7.1
+          env: SYMFONY_VERSION=4.0.x stability=beta
 
 before_install:
   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then echo 'memory_limit=-1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony:$SYMFONY_VERSION; fi
 
 install:
+  - if [ "$stability" != "" ]; then composer config minimum-stability $stability; endif
```
Comment (xabbuh) [inline .travis.yml]: I am sorry. I made a mistake in my comment. `endif` should be `fi`.

Comment (xabbuh) [reply .travis.yml]: Oh and this should probably also better be moved to the `before_install` step.

File: composer.json [hunk 3]
```diff
@@ -11,17 +11,17 @@
...
         "php": ">=5.3.3",
         "giggsey/libphonenumber-for-php": "~5.7|~6.0|~7.0|~8.0",
-        "symfony/framework-bundle": "~2.1|~3.0"
+        "symfony/framework-bundle": "~2.1|~3.0|^4.0"
     },
     "require-dev": {
         "doctrine/doctrine-bundle": "~1.0",
-        "jms/serializer-bundle": "~0.11|~1.0",
+        "jms/serializer-bundle": "~0.11|~1.0|~2.0",
         "phpunit/phpunit": "~4.0",
```
Comment (alcaeus) [inline composer.json]: Looks like phpunit 4.x doesn't really like Symfony 4: https://travis-ci.org/misd-service-development/phone-number-bundle/jobs/303934028#L761. You can try adding `|^5.7` here to get the build working.

**Comment by rh389** [comment]
Looks good, thanks all

**Comment by netbull** [comment]
@rh389 when can we expect the new release?

**Comment by rh389** [comment]
@netbull Good question. I'd like to merge anything that's ready and doesn't break SF2.1, cut a 1.3.0 release, and then bump requirements to SF2.7+ and work on a 2.0.0 release. Unfortunately I don't have packagist-foo for this lib. 

@thewilkybarkid could you spare some time to help out and/or lend me some packagist powers?

**Pull request merged**